packages/design-system/src/components/Table/Table.stories.tsx

```diff
@@ -140,55 +138,48 @@ export const Default: Story = {
   },
 };
 
-export const StackableTable: Story = {
-  ...Default,
-  args: {
-    stackable: true,
-    stackableBreakpoint: 'md',
-  },
-};
-
-export const MultiHeaderTable: Story = {
-  render: function Component() {
-    return (
-      <Table>
-        <TableCaption>Household members</TableCaption>
-        <col />
-        <colgroup span={2} />
-        <colgroup span={2} />
-        <TableHead>
-          <TableRow>
-            <TableCell component="td" rowSpan={1} />
-            <TableCell component="th" colSpan={2} scope="colgroup">
-              Address
-            </TableCell>
-            <TableCell colSpan={2} scope="colgroup">
-              Employment
-            </TableCell>
-          </TableRow>
-          <TableRow>
-            {multiHeaderTableData.headings.map(({ displayName }) => (
-              <TableCell component="th" scope="col" key={displayName}>
-                {displayName}
-              </TableCell>
-            ))}
-          </TableRow>
-        </TableHead>
-        <TableBody>
-          {multiHeaderTableData.data.map((dataItem, index) => (
-            <TableRow key={index}>
-              {multiHeaderTableData.headings.map((heading) => (
-                <TableCell
-                  key={`${heading.displayName}-${dataItem[heading.displayName]}`}
-                  component={heading.bodyComponent || ('td' as any)}
-                >
-                  {dataItem[heading.propName]}
-                </TableCell>
-              ))}
-            </TableRow>
-          ))}
-        </TableBody>
-      </Table>
-    );
-  },
-};
+// This UI doesn't seem to work with stackable layout
+// export const MultiHeaderTable: Story = {
```

Comment:
Commented this out because I have questions... the multi-header table seems very different from the default story; it's almost like a native table example that also uses our components? I'm seeing a mix of things like `<col>` and `<TableRow>`... Is this more of a pattern than a component?

The other question is... now that we force the stackable layout for small breakpoints it kinda breaks this entire example. Looking at [PROD](https://design.cms.gov/storybook/?path=/story/components-table--multi-header-table&globals=theme:core) and I'm not entirely sure how this pattern could work with the stacked layout. 

What to do?